src/main/java/com/ibm/cloudant/kafka/common/utils/JavaCloudantUtil.java

```diff
@@ -116,6 +112,7 @@ public static JSONArray batchWrite(Map<String, String> props, JSONArray data)
 				result.put(jsonResult);
 			}
 		} catch (Exception e) {
+			LOG.error("Exception caught in batchWrite()", e);
```

Comment:
This may need to be revisited in another PR - the worrying thing is that we were just swallowing exceptions from the cloudant client which I had manage to trigger with a misconfigured test.

Reply:
Agreed, there needs to be a separate look at error handling to conform to the behaviours of the built-in `errors.tolerance=all` and `none` flags. (`all` implies silently ignore bad messages so I guess that's all we have right now!)